### PR TITLE
Increase the default value of region-schedule-limit

### PR DESCRIPTION
**Pull request by shafreeck**
Signed-off-by: Shafreeck Sea <shafreeck@gmail.com>

### What problem does this PR solve? 

After introducing `store-limit`, `region-schedule-limit` is not necessary to use to limit the balance speed, so we change it to a larger value to avoid affecting the balance speed.

### What is changed and how it works?

Change the default region-schedule-limit from 64 to 2048

### Check List 

Tests 

 - Unit test

Code changes

 - Has configuration change

Related changes

 - Need to update the documentation

### CODE REVIEW

File: conf/config.toml [hunk 1]
```diff
@@ -64,7 +64,7 @@ max-snapshot-count = 3
 max-pending-peer-count = 16
 max-store-down-time = "30m"
 leader-schedule-limit = 4
-region-schedule-limit = 64
+region-schedule-limit = 2048
```
Comment (nolouch) [inline conf/config.toml]: no need to change replica limit?

Comment (shafreeck) [reply conf/config.toml]: I think replica-schedule-limit could be ajusted in another PR after doing some investitation.

Comment (rleungx) [reply conf/config.toml]: Is there possible to remove it?

Comment (shafreeck) [reply conf/config.toml]: The max number of concurrent operators should be in control, I think it is not necessary to remove it.

**Pull request merged**